src/main/java/com/google/sps/classes/MentorEvidence.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+/**
+ * The data used to authenticate a user.
+ */
+public final class MentorEvidence {
+
+  private final boolean isApprover; // This is to check whether the approver is assigned to mentor.
```

Comment:
When is this false?

Reply:
It's true when the approver is assigned to the mentor, and false when otherwise. I updated the comment to `This is true if the approver is assigned to the mentor.` to clarify. Is that good?

Reply:
I should probably understand this, but I don't think I do at the moment. When is MentorEvidence fetched for a user who isn't an approver of the mentor associated with that evidence??